src/main/kotlin/main.kt

```diff
@@ -0,0 +1,21 @@
+import domain.CoordinateRandomGenerator
+import domain.LandMineRandomMap
+import dto.CoordinateGeneratorRequest
+import dto.LandMineMapRequest
+import dto.LandMineMapUIRequest
+import view.InputView
+import view.LandMineMapUI
+
+fun main() {
+    val height = InputView.height()
+    val width = InputView.width()
+    val landmine = InputView.landmine()
+
+    val landMineMapRequest = LandMineMapRequest.of(height, width)
+    val coordinateGeneratorRequest = CoordinateGeneratorRequest.of(landMineMapRequest, landmine)
+    val landMineRandomMap = LandMineRandomMap(landMineMapRequest, CoordinateRandomGenerator(coordinateGeneratorRequest))
+
+    val mapUI = LandMineMapUI(LandMineMapUIRequest(landMineRandomMap))
+    mapUI.title()
+    mapUI.map()
```

Comment:
지뢰찾기 다른환경(ex. web)에서 동일하게 수행하려면 지금 작성하신 일련의 절차적인 코드를 동일하게 작성해야하는 상황이 발생할 수 있을것 같습니다.
이를 아래와 같이 추상화하여 관리해보시는건 어떨까요?
```suggestion
    MineSweeper.run(inputView, outputView)
```